### ISSUE TITLE
why you applied vq losses here?

### ISSUE THREAD

**Issue by primepake** [innnky/descript-audio-vae#4]
Thanks for great work! why you apply VQ losses here? it will help the model training more stable or something?

**Comment by rishikksh20** [comment]
@innnky @primepake I have same question, do VQ loss required ? Is it better to use pre-train DAC weights to initialize Encoder and Decoder weights then finetune it as VAE with VQ module ?

**Comment by primepake** [comment]
it doesn't need VQ loss, I trained 24k version here: https://github.com/primepake/learnable-speech/releases/tag/dac-vae
